src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -84,35 +84,21 @@ extension ExposureNotificationSettingViewController {
 	}
 
 	private func handleEnableError(_ error: ExposureNotificationError, alert: Bool) {
-		switch error {
-		case .exposureNotificationAuthorization:
-			logError(message: "Failed to enable exposureNotificationAuthorization")
-			if alert {
-				alertError(message: "Failed to enable: exposureNotificationAuthorization", title: AppStrings.ExposureSubmission.generalErrorTitle)
+		if alert {
+			switch error {
+			case .exposureNotificationAuthorization:
+				alertError(message: AppStrings.ExposureNotificationError.enAuthorizationError, title: AppStrings.ExposureNotificationError.generalErrorTitle, optinActions: [UIAlertAction(title: AppStrings.ExposureNotificationError.learnMoreActionTitle, style: .default, handler: { _ in WebPageHelper.showWebPage(from: self, urlString: AppStrings.ExposureNotificationError.learnMoreURL) })])
```

Comment:
From what I can see here, there is a lot of code duplication going on. I think this can be simplified. 👍😊

Reply:
done